platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/components/DefaultSavedStateController.java

```diff
@@ -55,20 +56,21 @@ public DefaultSavedStateController(@NonNull final StateConfig stateConfig) {
         this.stateConfig = Objects.requireNonNull(stateConfig);
     }
 
+    // TODO revisit synchronized keywords
```

Comment:
TODO

Reply:
Removed synchronized keywords (and TODO) since I don't think they are necessary for thread safety.